Makefile

```
@@ -23,5 +29,24 @@ ifeq "$(run_tests)" "yes"
 endif
 	CGO_ENABLED=0 go build -o $(OUTPUT)
 
+clean: SHELL:=/bin/bash
 clean:
```

checkmake (0.2.2)

[warning] Target body for "clean" exceeds allowed length of 5 (6).

(maxbodylength)